tests/unit/__snapshots__/TmBlock.spec.js.snap

```diff
@@ -383,7 +383,8 @@ exports[`TmBlock.vue has the expected html structure 1`] = `
                 <p class="tm-li-tx__content__caption__title">
                   <div>
                     Unbonded&nbsp;
-                    <b>105</b><span>&nbsp;ATOMS</span><span>&nbsp;- a few seconds ago</span>
+                    <b>100</b><span>&nbsp;Shares</span>
```

Comment:
i thought we were going to calculate the value of the shares in Atoms?

Reply:
This is fallback logic. Atoms need to be calculated and passed in. But the transaction component also works with the plain TX.